.eslintrc.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": ["semistandard", "plugin:jsdoc/recommended"],
```

Comment:
Since this repo is only going to use NodeJS we should try and use a more robust standard such as:
https://github.com/airbnb/javascript

We can discuss more about this if needed.

Reply:
Yep good idea, we used this before I remember. But lets do this in the next iteration of the repo.